omod/src/main/webapp/fragments/clinicalNotes.gsp

```diff
@@ -441,18 +441,33 @@ ${ ui.includeFragment("patientdashboardapp", "patientDashboardAppScripts", [note
 					</div>
 				</div>				
 			</div>			
-
+		
 			<div id="confirmationQuestion">
 				<p style="display: inline;">
 					<button class="submitButton confirm" style="float: right">Submit</button>
 				</p>
 
-				<button class="cancel" style="margin-left: 5px;">Cancel</button>
+				<button id="cancelButton" class="cancel cancelButton" style="margin-left: 5px;">Cancel</button>
 				<p style="display: inline">&nbsp;</p>
 			</div>
 		</div>
     </form>
 </div>
+<div id="confirmDialog" class="dialog" style="display: none;">
+        <div class="dialog-header">
+            <i class="icon-save"></i>
+
+            <h3>Confirm</h3>
+        </div>
+
+
+        <div class="dialog-content">
+            <h3>Cancelling will lead to loss of data,are you sure you want to do this?</h3>
+
+            <span id="cancelButton" class="button confirm right" style="float: right">Confrim</span>
```

Comment:
@kimani-njoroge, why have two  similar `ids (cancelButton)` on the same page? Can you use `class` instead?

/cc @daugmaud